Khan/perseus
# [Interactive Graph Editor] Implement the UI for adding start coords for circle graphs

## Summary:
Add the UI to specify start coords for Circle graph type.

- Add the circle graph type to `start-coor-settings.tsx`
- Create a `start-coords-circle.tsx` file with the main UI
- Update the circle graph type so that `startCoords` includes both
  the center and the radius.
- Update tests

Issue: https://khanacademy.atlassian.net/browse/LEMS-2206

## Test plan:
`yarn jest`

Storybook
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--interactive-graph-circle-with-starting-coords
- Confirm that the preview shows the circle at the already set coords + radius
- Confirm that the UI reflects these coords + radius
- Press "Use default start coordinates"
- Confirm that the circle goes to [0, 0] with a radius of 2.

- Go to http://localhost:6006/?path=/story/perseuseditor-editorpage--demo
- Create an interactive graph widget
- Change the graph to circle type
- Confirm that the start coords UI shows the center at [0, 0] with radius 2

<img width="384" alt="Screenshot 2024-07-25 at 4 29 34 PM" src="https://github.com/user-attachments/assets/8756f8f9-6a4c-40c0-b609-de10d7844612">
